CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+2.6.0 (In Progress)
+===================
```

Comment:
Did you want to add changes of this PR?

Reply:
Not yet. We will add changelog before release, after we finish all improvements. Currently, it's not clear what other improvements will go in 2.6 since it's a moving target.